src/utils.rs

```diff
@@ -80,7 +80,41 @@ pub fn ident_at(root: &SyntaxNode, offset: usize) -> Option<CursorInfo> {
         }
     }?;
     let parent = ident.node().parent();
-    if let Some(attr) = parent.clone().and_then(Key::cast) {
+    if parent.clone().map(|x| x.kind() == rnix::SyntaxKind::NODE_INHERIT).unwrap_or(false) {
```

Comment:
Question: Do we have to `.clone()` here? Seems like a reference should be enough?

Oh and

```suggestion
    if parent.clone().map_or(false, |x| x.kind() == rnix::SyntaxKind::NODE_INHERIT) {
```

I've realised recently is cleaner

Reply:
fixed :)